tasks/populate-release-notes-images/README.md

```diff
@@ -2,6 +2,7 @@
 
 Tekton task to populate the releaseNotes.content.images key in the data.json file. It will update the data.json
 in place so that downstream tasks relying on the releaseNotes data can use it.
+Additionally, it outputs a result with data used in component SBOM generation.
```

Comment:
Can you reword this to make it clear that it's a path to a file with the data and not the actual data in the result? (Same applies to the change log entry.)